Dockerfile

```diff
@@ -10,4 +11,4 @@ RUN pipenv install --deploy --ignore-pipfile
 
 EXPOSE 8000
 
-CMD ["pipenv", "run", "gunicorn", "-w 2", "-b 0.0.0.0:8000", "-k uvicorn.workers.UvicornWorker", "api.main:app"]
\ No newline at end of file
+CMD ["sh", "/app/entrypoint", "pipenv", "run", "gunicorn", "-w 2", "-b 0.0.0.0:8000", "-k uvicorn.workers.UvicornWorker", "api.main:app"]
```

Comment:
How about:
`CMD ["pipenv", "run", "sh", "/app/entrypoint", "gunicorn", "-w 2", "-b 0.0.0.0:8000", "-k uvicorn.workers.UvicornWorker", "api.main:app"]`
then we don't need 
`RUN pip install kafka-python`